x-pack/plugins/gis/public/shared/layers/tile_layer.js

```diff
@@ -30,14 +31,52 @@ export class TileLayer extends AbstractLayer {
     return tileLayerDescriptor;
   }
 
+  _tileLoadErrorTracker(map, url) {
+    let tileLoad;
+    map.on('dataloading', ({ tile }) => {
+      if (tile && tile.request) {
+        // If at least one tile loads, endpoint/resource is valid
+        tile.request.onloadend = ({ loaded }) => loaded && (tileLoad = true);
```

Comment:
`tile.request.onloadend = ({ loaded }) => loaded && (tileLoad = true);` is difficult to read in a single line. How about breaking it out so its easier to read and under stand that tileLoad is getting set to true

```
        tile.request.onloadend = ({ loaded }) => {
          if (loaded) {
            tileLoad = true;
          }
        };
```